scripts/load_test_customer_data.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+# This script assumes you have the nise command installed in your environment
+# See: https://pypi.org/project/koku-nise/
+# It also assumes you have the repo locally with example files:
+# See: https://github.com/project-koku/nise
+# Once a local copy of the repo is available set it's path using the
+# NISE_REPO_PATH environment variable
+
+# Assumes environment variables have been set:
+#   - POSTGRES_SQL_SERVICE_PORT
+#   - POSTGRES_SQL_SERVICE_HOST
+#   - DATABASE_ADMIN (postgres admin user)
+#   - DATABASE_PASSWORD (postgres admin user's password)
+#   - DATABASE_USER (postgres user to be recreated)
+
+export PGPASSWORD="${DATABASE_PASSWORD}"
+export PGPORT="${POSTGRES_SQL_SERVICE_PORT}"
+export PGHOST="${POSTGRES_SQL_SERVICE_HOST}"
+export PGUSER="${DATABASE_ADMIN}"
+
+KOKU_PATH=$1
+START_DATE=$2
+END_DATE=$3
+
+if [ -z "$NISE_REPO_PATH" ]; then
+  echo "Please set NISE_REPO_PATH in your env"
+  exit
+fi
+
+CHECK=$(curl -s -w "%{http_code}\n" -L "localhost:${KOKU_PORT}$API_PATH_PREFIX/v1/status/" -o /dev/null)
+if [[ $CHECK != 200 ]];then
+    echo "Koku server is not available at localhost:${KOKU_PORT}. Exiting."
+    exit 0
+fi
+
+CHECK=$(curl -s -w "%{http_code}\n" -L "localhost:${MASU_PORT}$API_PATH_PREFIX/v1/status/" -o /dev/null)
+if [[ $CHECK != 200 ]];then
+    echo "Masu server is not available at localhost:${MASU_PORT}. Exiting."
+    exit 0
+fi
+
+# Set our date ranges
+sed -i'.example'  -e "s/2019-04-01/$START_DATE/g" -e "s/2019-07-31/$END_DATE/g" "$NISE_REPO_PATH/examples/ocp_on_aws/aws_static_data.yml"
```

Comment:
So, I am about to break you here with my documentation pr in nise. 
https://github.com/project-koku/nise/pull/192/files#diff-e7a997a65c3defbc3edc971db29dc25cR4

However, I can easily roll those changes back. I just ran into problems with masu expecting the current month when I was trying to figure out how to create an ocp on aws provider.

Reply:
I can find/replace any string as long as the file has a start date and end date.

Reply:
Sweet. I think I will roll my changes back anyway to not break you. I think everyone will be using this script in the future anyway.